style.css

```diff
@@ -0,0 +1,147 @@
+@font-face {
+    font-family: 'NeueMontrealBook';
+    src: url('PPNeueMontreal-Book.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealBook';
+    src: url('PPNeueMontreal-Book.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealBook';
+    src: url('PPNeueMontreal-Book.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealItalic';
+    src: url('PPNeueMontreal-Italic.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealMedium';
+    src: url('PPNeueMontreal-Medium.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealSemiBoldItalic';
+    src: url('PPNeueMontreal-SemiBolditalic.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealThin';
+    src: url('PPNeueMontreal-Thin.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'GoshaSansBold';
+    src: url('PPGoshaSans-Bold.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'GoshaSansRegular';
+    src: url('PPGoshaSans-Regular.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+.top_container {
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    position: relative;
+}
+
+header {
+    font-family: GoshaSansBold, sans-serif;
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    background-color: #661D09;
+    color: #FFF;
+}
+
+h1 {
+    font-family: GoshaSansBold, sans-serif;
+}
+
+p {
+    font-family: GoshaSansBold, sans-serif;
+}
+
+.instructions_button {
+    font-family: GoshaSansBold, sans-serif;
+}
+
+button {
+    font-family: GoshaSansBold, sans-serif;
+    width: 5rem;
+    border: solid black 1px;
+    border-radius: 5px;
+    background-color: #FFFFFF;
+    padding: .5rem;
+}
+
```

Comment:
this will likely style all buttons in app by default now to this styling. Is this the intention? Can we select via a class?

Reply:
Same is true for all selectors above that are being selected via tag as opposed to class.